main.py

```diff
@@ -59,13 +59,22 @@ def main():
             data_args.train_dataset_fraction,
         )
 
-    if training_args.do_eval:
+    if training_args.do_train and training_args.do_eval:
         logger.info(f"Loading evaluation dataset from {data_args.dataset_config_path}")
-        raw_datasets["eval"] = load_datasets_from_config(
+        raw_datasets["val"] = load_datasets_from_config(
             data_args.dataset_config_path,
-            "eval",
+            "val",
             16000,
-            data_args.eval_dataset_fraction,
+            data_args.train_dataset_fraction,
```

Comment:
Should we use `data_args.eval_dataset_fraction` instead?

Reply:
I need to clean this up, but no, this is an intentional change! I'll reword this in a future PR that touches main.py so it's not misleading. Thanks for flagging it!